server/test/unit/server/db/test_model.py

```diff
@@ -590,3 +590,30 @@ def test_meta_collection(self):
         Assert that the collection name is correct.
         """
         self.assertEquals(model.CeleryBeatLock._meta['collection'], 'celery_beat_lock')
+
+
+class TestDeferredDownload(unittest.TestCase):
+    """
+    Test the DeferredDownload class.
+    """
+
+    def test_model_superclass(self):
+        sample_model = model.DeferredDownload()
+        self.assertTrue(isinstance(sample_model, Document))
+
+    def test_attributes(self):
+        self.assertTrue(isinstance(model.DeferredDownload.unit_id, StringField))
+        self.assertTrue(model.DeferredDownload.unit_id.required)
+
+        self.assertTrue(isinstance(model.DeferredDownload.unit_type_id, StringField))
+        self.assertTrue(model.DeferredDownload.unit_type_id.required)
+        self.assertEqual(['unit_id'], model.DeferredDownload.unit_type_id.unique_with)
+
+        self.assertTrue(isinstance(model.DeferredDownload._ns, StringField))
+        self.assertEqual('deferred_download', model.DeferredDownload._ns.default)
+
+    def test_meta_collection(self):
+        """
+        Assert that the collection name is correct.
+        """
+        self.assertEquals(model.DeferredDownload._meta['collection'], 'deferred_download')
```

Comment:
@jortel I verified that `meta` is removed and `_meta` is added. Here's my snippet.

```
from mongoengine import Document, StringField

class DeferredDownload(Document):
    meta = {
        'collection': 'deferred_download',
        'indexes': [
            {
                'fields': ['unit_id', 'unit_type_id'],
                'unique': True
            }
        ]
    }

    unit_id = StringField(required=True)
    unit_type_id = StringField(required=True)

    # For backward compatibility
    _ns = StringField(default='deferred_download')


print DeferredDownload.meta
```

When I run that snippet it outputs:

```
Traceback (most recent call last):
  File "/tmp/foo.py", line 21, in <module>
    print DeferredDownload.meta
AttributeError: type object 'DeferredDownload' has no attribute 'meta'
```